crates/cli/src/cli/mod.rs

```diff
@@ -169,7 +169,7 @@ pub struct StartSimnet {
     #[arg(long = "airdrop-amount", short = 'q', default_value = DEFAULT_AIRDROP_AMOUNT)]
     pub airdrop_token_amount: u64,
     /// List of keypair paths to airdrop
-    #[arg(long = "airdrop-keypair-path", short = 'k', default_value = DEFAULT_SOLANA_KEYPAIR_PATH.as_str())]
```

Comment:
One thin this default being part of this clap struct gave us is the default being displayed in our help text:

<img width="547" height="152" alt="Image" src="https://github.com/user-attachments/assets/268791d1-b1cb-43fa-9fa3-ccb224592066" />

Can we update the comment to show the default now that we've removed it from here?